src/CommunityToolkit.Maui.MediaElement/Views/MediaManager.android.cs

```diff
@@ -262,7 +266,54 @@ public void OnVolumeChanged(float volume)
 
 		MediaElement.Position = TimeSpan.Zero;
 	}
+	protected virtual partial void PlatformFullScreen()
+	{
+		var activity = Platform.CurrentActivity;
```

Comment:
can this part be extracted to separate method to avoid code duplication (lines 271 - 293 and lines 297 -315

Reply:
Sure sounds like a good idea.

Reply:
Extracted method and removed code duplication.